### PR TITLE
Cx v0.7

### LINKED ISSUE #23: Update skycoin-cx formula for release 0.7.0
Issue tracker is **ONLY** used for reporting bugs, features and enhancements. NO SUPPORT TICKETS ACCEPTED! Use [stackoverflow](https://stackoverflow.com) for supporting issues.

**Is your feature request related to a problem? Please describe.**
No

**Describe the solution you'd like**
Add support for CX v0.7.0

**Describe alternatives you've considered**
NA

**Additional context**
I will raise a PR shortly to address this

### PR DESCRIPTION
**Pull request by BigOokie**
Fixes #23

Changes:
- Updated Skycoin-cx formula for v0.7.0

Does this change need to mentioned in CHANGELOG.md?
Yes. Included in PR

### CODE REVIEW

**Comment by BigOokie** [comment]
@olemis Can you take a look at this when you have a chance, and after Travis build finished. Not urgent

File: Formula/skycoin-cx.rb [hunk 1]
```diff
@@ -23,13 +23,15 @@
...
-  sha256 "92c6badd27c675bd4b4c7a2db68a22cdf4b0ea3dfa12ea75c7322e3c47420b25"
+  url "https://github.com/skycoin/cx/releases/download/v0.7.0/cx-0.7.0-bin-macos-x64.zip"
+  version "0.7.0"
+  sha256 "20209b50054e453d87f738225f9d3830d89884366a7c391fdbbc5dfc3fe85ee8"
 
   bottle :unneeded
 
   def install
-    bin.install "cx"
+    ## v0.7.0 has a different folder structure in the zip package.
```
Comment (olemis) [inline Formula/skycoin-cx.rb]: really need to include this in formula ?

Comment (BigOokie) [reply Formula/skycoin-cx.rb]: Thx for the feedback. Will update now

**Comment by BigOokie** [comment]
@olemis I just pushed the change you suggested. Once Travis has finished it should be good to go. Thanks again

**Pull request merged**